tests/vectordb/test_lancedb.py

```diff
@@ -0,0 +1,211 @@
+import shutil
+import pytest
+
+from embedchain import App
+from embedchain.config import AppConfig
+from embedchain.config.vectordb.lancedb import LanceDBConfig
+from embedchain.vectordb.lancedb import LanceDB
+
+
+@pytest.fixture
+def lancedb():
+    return LanceDB(config=LanceDBConfig(dir="test-db", collection_name="test-coll"))
+
+
+@pytest.fixture
+def app_with_settings():
+    lancedb_config = LanceDBConfig(allow_reset=True, dir="test-db-reset")
+    lancedb = LanceDB(config=lancedb_config)
+    app_config = AppConfig(collect_metrics=False)
+    return App(config=app_config, db=lancedb)
+
+
+@pytest.fixture(scope="session", autouse=True)
+def cleanup_db():
+    yield
+    try:
+        shutil.rmtree("test-db.lance")
+    except OSError as e:
+        print("Error: %s - %s." % (e.filename, e.strerror))
+
+
+@pytest.mark.skip(reason="Logging setup needs to be fixed to make this test to work")
```

Comment:
Please take reference from [here](https://github.com/embedchain/embedchain/commit/827d63d115d32d573cdcee8bbd55479349474df5#diff-f00fe2099a765dd149fafcd0022fdff1c3064c8aa3acab619d2b087b86f8510aL58) and try to make this test pass

Reply:
sure